src/licensedcode/data/rules/apache-2.0_893.yml

```diff
@@ -1,5 +1,5 @@
 license_expression: apache-2.0
-is_license_notice: yes
+is_license_reference: yes
```

Comment:
`The Apache License, Version 2.0 https://www.apache.org/licenses/LICENSE-2.0.txt`